extension/index.html

```diff
@@ -0,0 +1,18 @@
+<html>
+  <head>
+    <style>
+      body {
+        margin: 0;
+      }
+
+      iframe {
+        border: none;
+        height: 600px; /* Same height defined in src/App.js */
+        width: 400px; /* Same width defined in src/App.js */
+      }
+    </style>
+  </head>
+  <body>
+    <iframe src="http://localhost:3000"></iframe>
```

Comment:
This should be a dynamically constructed url based on environment. Unsure best way to go about that in this build.